usage-advanced.py

```diff
@@ -2,226 +2,215 @@
+                    ],
+                    value=["controls"],
+                ),
+                html.P("Volume:", style={"margin-top": "10px"}),
+                dcc.Slider(
+                    id="slider-volume",
+                    min=0,
+                    max=1,
+                    step=0.05,
+                    value=None,
+                    updatemode="drag",
+                    marks={0: "0%", 1: "100%"},
+                ),
+                html.P("Playback Rate:", style={"margin-top": "25px"}),
+                dcc.Slider(
+                    id="slider-playback-rate",
+                    min=0,
+                    max=4,
+                    step=None,
+                    updatemode="drag",
+                    marks={i: str(i) + "x" for i in [0, 0.25, 0.5, 0.75, 1, 2, 3, 4]},
+                    value=1,
+                ),
+                html.P(
+                    "Update Interval for Current Time:", style={"margin-top": "30px"}
+                ),
+                dcc.Slider(
+                    id="slider-intervalCurrentTime",
+                    min=40,
+                    max=1000,
+                    step=None,
+                    updatemode="drag",
+                    marks={i: str(i) for i in [40, 100, 200, 500, 1000]},
+                    value=100,
+                ),
+                html.P(
+                    "Update Interval for seconds loaded:", style={"margin-top": "30px"}
+                ),
+                dcc.Slider(
+                    id="slider-intervalSecondsLoaded",
+                    min=200,
+                    max=2000,
+                    step=None,
+                    updatemode="drag",
+                    marks={i: str(i) for i in [200, 500, 750, 1000, 2000]},
+                    value=500,
+                ),
+                html.P("Update Interval for duration:", style={"margin-top": "30px"}),
+                dcc.Slider(
+                    id="slider-intervalDuration",
+                    min=200,
+                    max=2000,
+                    step=None,
+                    updatemode="drag",
+                    marks={i: str(i) for i in [200, 500, 750, 1000, 2000]},
+                    value=500,
+                ),
+                html.P(
+                    "Seek To (% of total media length):", style={"margin-top": "30px"}
+                ),
+                dcc.Slider(
+                    id="slider-seek-to",
+                    min=0,
+                    max=1,
+                    step=None,
+                    updatemode="drag",
+                    marks={i: str(i * 100) + "%" for i in [0, 0.25, 0.5, 0.75, 1]},
+                    value=0,
+                ),
+            ],
+        ),
+    ]
+)
+
+
+@app.callback(
+    Output("video-player", "url"),
+    [Input("button-update-url", "n_clicks")],
+    [State("input-url", "value")],
+    prevent_initial_call=True,
+)
```

Comment:
```suggestion
@callback(
    Output("video-player", "url"),
    Input("button-update-url", "n_clicks"),
    State("input-url", "value"),
    prevent_initial_call=True,
)
```
Doesn't have to be in this PR, but at some point the rest of this syntax should be updated as well :)